docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -12,173 +12,40 @@ videoBanner: UFhT52d5bYg
 - AWS account with a Redshift cluster and permissions to create and attach IAM
   policies.
 
-## Step 1/5. Install Teleport
+## Step 1/6. Install Teleport
 
 (!docs/pages/includes/database-access/start-auth-proxy.mdx!)
 
-## Step 2/5. Create Teleport user
+## Step 2/6. Create Teleport user
 
 (!docs/pages/includes/database-access/create-user.mdx!)
 
-## Step 3/5. Configure IAM
+## Step 3/6. Create database agent configuration
```

Comment:
When are we using "Database Service" vs "database agent"? Are the two terms interchangeable?

Reply:
I'm not sure which one we should use or if we can use both. @r0mant, do you have any thoughts on this?

Reply:
On second thought, using "database agent" vs. "Database Service" is a docs-wide issue, so we can talk about it a separate issue so we don't block this PR.

Reply:
Yeah, I've been trying to standardize on "Database Service" or "Database Service agent" following @ptgott's lead.